flang/lib/Lower/Bridge.cpp

```diff
@@ -657,12 +657,9 @@ class FirConverter : public Fortran::lower::AbstractConverter {
     };
     // Add labels from an explicit list.  The list may have duplicates.
     for (auto &label : std::get<std::list<Fortran::parser::Label>>(stmt.t)) {
-      if (labelSet.count(label) == 0) {
-        // Ignore labels with no ASSIGN statements for the selector variable.
-        continue;
-      }
-      if (std::find(indexList.begin(), indexList.end(), label) ==
-          indexList.end()) { // ignore duplicates
+      if (labelSet.count(label) &&
```

Comment:
Does `count` return the number of duplicates of label? Is there an exists in set test?

Reply:
llvm/include/llvm/ADT/SmallSet.h says 
`/// count - Return 1 if the element is in the set, 0 otherwise.
`
I don't find any other alternative for membership queries, which is what this is.  (No 'contains', for example.)  The name 'count' may come from an attempt to use uniform names for a variety of container class accesses.